dist/templates/ovn-setup.yaml.j2

```diff
@@ -130,8 +130,9 @@ rules:
 - apiGroups:
   - k8s.cni.cncf.io
   resources:
-  - ipamleases 
-  verbs: ["create", "list", "get", "watch"]
+  - ipamleases
+  - ipamleases/status
+  verbs: ["create", "list", "get", "watch", "update"]
```

Comment:
we only need "update" on the status sub-resource, and we do not need create on the "ipamleases".